fineract-core/src/main/java/org/apache/fineract/useradministration/domain/Permission.java

```diff
@@ -43,12 +43,16 @@ public class Permission extends AbstractPersistableCustom<Long> implements Seria
     @Column(name = "can_maker_checker", nullable = false)
     private boolean canMakerChecker;
 
+    @Column(name = "is_replayable", nullable = false)
+    private boolean isReplayable;
```

Comment:
Can you please elaborate on this flag? I am not fully understanding the business logic here